.github/ISSUE_TEMPLATE/6-AZ-DOC-FEEDBACK-CONTROL.yml

```
@@ -0,0 +1,67 @@
+name: "📄 Feedback control template for docs"
+title: "Feedback"
+description: >-
+  ⛔ This template is intended for use by the feedback control on the bottom of every article page on the docs site. If you aren't using the feedback control, choose one of the other templates.⛔
+labels:
+  - "needs-triage"
```

Might want to include the "Doc - Reference" label as well.
```suggestion
  - "needs-triage,Doc - Reference"
```

---

The labels match the existing docs issue template.